### PR TITLE
docs: clear all filters + updated screenshots

### PR DESCRIPTION
**Pull request by NickIliev**
Related to https://kinvey.atlassian.net/browse/FID-1493

### CODE REVIEW

File: user-guide/live-traffic/live-traffic.md [hunk 1]
```diff
@@ -9,116 +9,133 @@ previous_url: /user-guide/toolbar/decode-content, /user-guide/toolbar
 
 ## Live Traffic Tab
 
-The __Live Traffic__ tab is one of the most important feature of Fiddler Everywhere. It display a short summary of each captured session (in the __Live Traffic list__) and provides functionalities to work with these sessions (via the __Live Traffic toolbar__ and the Inspectors tabs). Fiddler Everywhere enables sessions saving and sharing.
+The __Live Traffic__ tab is  is an essential feature of Fiddler Everywhere. It display a summary of each captured session (in the [**Live Traffic list**](#live-traffic-list)) and provides functionalities to work with these sessions (via the [**Live Traffic toolbar**](#live-traffic-toolbar), the [**Inspectors**]({%slug inspector-types%}) tabs, and the interconnected [**Auto Responder**]({%slug auto-responder-general%}). The section also enables sessions saving and sharing, editing issued requests, marking, commeting sessions, and applying Auto Responder rules.
```
Comment (rosen-vladimirov) [inline user-guide/live-traffic/live-traffic.md:12]: it display -> it display**s** maybe?

File: user-guide/live-traffic/live-traffic.md [hunk 2]
```diff
@@ -127,32 +144,36 @@ Many operations begin by selecting one or more entries in the Live Traffic and t
...
-- __Method__ - The HTTP Method used by the request (GET, POST, etc.)
+- __Method__ - The HTTP Method used by the made request (GET, POST, PUT, etc.)
 - __Body__ - The number of bytes in the response body
 - __Caching__ - Values from the Response's Expires and Cache-Control headers
 - __Content-Type__ The Content-Type header from the response
 - __Process__ The local OS process from which the traffic originated
-- __Comments__ Field for showing the custom comments added by you (or the author of a shared session).
+- __Comments__ A field for showing the custom comments added by you (or the author of a shared session).
+
+Each column comes with a **more** button (three vertical dots), which opens a **Filter** popup. Use this functionality to apply filters for the currently selected column only (e.g., filter all the traffic by _host_ name or by a specific _status code_). Columns with active filters will have a blue filter indicator. You can clear active filters via the **Filter** popup (for the filtered column) or use the global [__Clear All Filters__ button](#clear-all-filters).
```
Comment (rosen-vladimirov) [inline user-guide/live-traffic/live-traffic.md:163]: I'm not sure if it is the correct place, but it is worth mentioning that with this three-dots menu users are able to select which columns to see - just click on the three dots and then on the Columns in the popup - you'll have checkboxes for each available column and you can select which one you want to see.

File: user-guide/live-traffic/live-traffic.md [hunk 3]
```diff
@@ -164,7 +185,7 @@ The context menu option __Save__ provides the means to save selected sessions. T
 
 All three options will display a save prompt.
 
-1.  Select the sessions, open the context menu, and click on __Save__ option.
+1.  Select the sessions, open the context menu, and click on the __Save__ option.
```
Comment (rosen-vladimirov) [inline user-guide/live-traffic/live-traffic.md:188]: I think there's one additional space in the beginning, which should be removed.

File: user-guide/live-traffic/live-traffic.md [hunk 4]
```diff
@@ -218,15 +239,15 @@ Use the __Replay__ context menu option to replay previously captured sessions.
 
 ### Select
 
-Use the __Select__ context menu option to select sessions that are directly related to the currently selected session. The options are to select __parent__, __children__, and __duplicate__ sessions.
+Use the __Select__ context menu option to select sessions that are directly related to the currently selected session.  The possibilities are to choose  __parent__, __children__, and __duplicate__ sessions.
```
Comment (rosen-vladimirov) [inline user-guide/live-traffic/live-traffic.md:242]: Additional space is added before `The possibilities`, which shouldn't be there. Also additional space is added after choose and before `__parent__` which probably shouldn't be there as well.

File: user-guide/live-traffic/live-traffic.md [hunk 5]
```diff
@@ -218,15 +239,15 @@ Use the __Replay__ context menu option to replay previously captured sessions.
...
 
 - __Parent request__ Selects the session that is a parent of the currently selected session. Keyboard shortcut: __P__
-- __Children requests__ Selects the sessions that are children of the currently selected session. Keyboard shortcut: __C__
+- __Children requests__ Select the sessions that are children of the currently selected session. Keyboard shortcut: __C__
```
Comment (rosen-vladimirov) [inline user-guide/live-traffic/live-traffic.md:245]: I'm not sure this change is correct - we are talking about context menu item here, so probably it should be Selects. Also it is inconsistent with the rest of the items here.

**Review by rosen-vladimirov — APPROVED** [review]
Some minor comments added, overall looks good.

File: user-guide/live-traffic/live-traffic.md (hunk 5 above)
Comment (NickIliev) [reply user-guide/live-traffic/live-traffic.md:245]: Its a bit confusing but the change is actually grammatically correct it is **Selects the session** (singular) VS **Select the sessions** (plural). This option (_Children requests_) will select multiple sessions and the other are options for singular selection.

**Pull request merged**